lua/mason-registry/jqls/init.lua

```diff
@@ -0,0 +1,15 @@
+local Pkg = require "mason-core.package"
+local _ = require "mason-core.functional"
+local go = require "mason-core.managers.go"
+
+return Pkg.new {
+    name = "jqls",
+    desc = _.dedent [[
+        jqls is a language server for the jq language, developed by Matthias Wader. It provides
```

Comment:
"Matthias Wade" is a cool name but sadly my real name is "Mattias Wadman" :)

Reply:
Sorry!!

Reply:
No worries :)

Reply:
It was the mwader twitter handle that threw me off

Reply:
Aha yeah someone already had @wader :(